src/backend/protocols/go.twitter/worker.go

```diff
@@ -247,9 +269,11 @@ func (w *Worker) RegisterAccountHandler(accountHandler *AccountHandler) {
 }
 
 func (w *Worker) RemoveAccountHandler(accountHandler *AccountHandler) {
-	workerKey := accountHandler.userAccount.userID.String() + accountHandler.userAccount.remoteID.String()
-	w.WorkersGuard.Lock()
-	accountHandler.Stop(true)
-	delete(w.AccountHandlers, workerKey)
-	w.WorkersGuard.Unlock()
+	if accountHandler != nil {
```

Comment:
We should not call this function with a nil accountHandler or it's not a normal situation if it can occurs and a warning log could be useful to see this situation

Reply:
Difficult to know when this could occur, but it occurred sometime during my test ! Indeed, there are many ways to stop accountHandler (closing chan, sending stop orders, suicide on errors, etc.)…

Anyway, I added a check before calling RemoveAccountHandler.